### PR TITLE
SP11/SP1/5 | Verify user cannot activate Trial period with invalid CVV

### PR DESCRIPTION
**Pull request by SvetlanaKanel**
https://signjstest.atlassian.net/jira/software/projects/SP/boards/1?selectedIssue=SP-1
CVV empty 
CVV with1 digit
CVV with 2 digits

format Registration -> add test.descrive for Business user

### CODE REVIEW

File: tests_advance/registration.spec.js [hunk 1]
```diff
@@ -11,11 +11,12 @@ import {
     JIRA_LINK,
     SUBSCRIBE_TO_BUSINESS_PLAN,
     INVALID_CARD_NUMBER,
-    ERROR_CARD_NUMBER_COLOR,
+    ERROR_TEXT_COLOR,
```
Comment (Elgrueber) [inline tests_advance/registration.spec.js:14]: Do all errors have same text color?

File: tests_advance/registration.spec.js [hunk 2]
```diff
@@ -11,11 +11,12 @@ import {
...
     NEGATIVE_CONFIRM_CODE,
     PLEASE_ENTER_CONFIRMATION_CODE,
     TOAST_MESSAGE,
     INVALID_CARD_EXPIRATION_DATE,
+    INVALID_CVV,
 } from '../testData';
 import { generateNewUserData, retrieveUserEmailConfirmCode } from '../helpers/utils';
 import { description, tags, severity, Severity, epic, step, tag, link, feature } from 'allure-js-commons';
```
Comment (Elgrueber) [inline tests_advance/registration.spec.js:22]: severity or Severity?

File: tests_advance/registration.spec.js (hunk 1 above)
Comment (SvetlanaKanel) [reply tests_advance/registration.spec.js:14]: all errors which I found in this suit had this color, but we have another error color in test's data.

File: tests_advance/registration.spec.js (hunk 2 above)
Comment (SvetlanaKanel) [reply tests_advance/registration.spec.js:22]: await severity(Severity.NORMAL);

File: tests_advance/registration.spec.js (hunk 1 above)
Comment (Elgrueber) [reply tests_advance/registration.spec.js:14]: Then could we change the name of the variable (like it was before, error_card_number_text_color? or error_CC_text_color, or error_credit_card_text_color? To differentiate the name of this error with others errors of text color in test data?

Comment (SvetlanaKanel) [reply tests_advance/registration.spec.js:14]: we have this one: ERROR_COLOR = 'rgb(255, 81, 81)' that doesn't work for my case, but all other my error messages have text with ERROR_TEXT_COLOR, so I am not sure that it is a good idea to add several constants with the same color and meaning and different names

Comment (SvetlanaKanel) [reply tests_advance/registration.spec.js:14]: Эту вторую переменную я завела и сначала назвала конкретно по имени поля, потому что мне не подошла та, которую уже кто-то завел раньше, но поскольку в моем случае все поля имеют один и тот же цвет, не имеет смысла делать разные константы для каждого поля.

**Review by Elgrueber — APPROVED** [review]

**Pull request merged**